cfpb/cfgov-refresh
# Updates helper text in reusable text module

Helper text was too long and was being truncated. This re-writes it to be shorter.

## Changes

- Shortens sidefoot heading helper text in reusable text snippet.

## Testing

1. Create a test Document detail page in Wagtail.
2. Click on the sidebar and add a Reusable Text.
3. Choose a Reusable Text, edit it to add "sidefoot heading" if absent (latest database dump appears not to have any reusable text snippets with a sidefoot heading set.
4. Check the helper text when hovering over the field.

## Screenshots

Before:
![screen shot 2017-07-24 at 4 58 24 pm](https://user-images.githubusercontent.com/704760/28545403-de16f02c-7094-11e7-9566-eab7baf597e1.png)

After (close-up):
![screen shot 2017-07-24 at 5 18 49 pm](https://user-images.githubusercontent.com/704760/28545402-de1249d2-7094-11e7-8559-f77f1b2b8350.png)
